src/base/zonefile_fmt.rs

```diff
@@ -338,7 +405,7 @@ mod test {
             more like a silly monkey with a typewriter accidentally writing \
             some shakespeare along the way but it feels like I have to type \
             e\" \"ven longer to hit that limit!\"",
-            record.display_zonefile(false).to_string()
+            record.display_zonefile(DisplayKind::Simple).to_string()
```

Comment:
Unrelated to this PR I like the text content above hehe, but shouldn't that be "Shakespeare" with a capital S? And sadly: https://edition.cnn.com/2024/11/01/science/monkeys-cannot-type-shakespeare-study-intl-scli-scn/index.html ;-)